packages/volto-slate/src/editor/SlateEditor.jsx

```diff
@@ -340,6 +341,7 @@ class SlateEditor extends Component {
                 onKeyDown && onKeyDown({ editor, event });
               }}
               {...editableProps}
+              aria-labelledby={`field-${id}`}
```

Comment:
@Wagner3UB This isn't the same as the change in #6772, which also uses `fieldSet`...what's the reason for the difference?

Reply:
@davisagli No, you are correct, it is the same. In #6772 there was an acceptance test failure, which I have already solved here. I’ll try to take a look today at all my new PRs.